spinalcordtoolbox/deepseg/models.py

```diff
@@ -186,7 +186,7 @@
          "url": [
              "https://github.com/sct-pipeline/fmri-segmentation/releases/download/v0.2/model-fmri-segmentation-v0.2.zip"
          ],
-         "description": "Spinal cord segmentation for EPI data using nnUNetv2 model",
+         "description": "Spinal cord segmentation for mean EPI data using nnUNetv2 model",
```

Comment:
```suggestion
         "description": "Spinal cord segmentation for EPI data (single 3D volume)",
```

data don't necessarily need to be average-- eg model works on single time point

Reply:
Indeed, thank you for pointing that out